tensorflow/lite/core/model_builder.h

```diff
@@ -155,6 +155,33 @@ class FlatBufferModel {
       const tflite::Model* caller_owned_model_spec,
       ErrorReporter* error_reporter = DefaultErrorReporter());
 
+#if FLATBUFFERS_LITTLEENDIAN == 0
+  /// Byte swap a constant buffer in place.
+  static void ByteSwapBuffer(int8_t tensor_type, size_t buffer_size,
+                             uint8_t* buffer);
+
+  /// Byte swap the buffers field of a TFLite Model instance in place.
+  static void ByteSwapTFLiteModel(const tflite::Model* tfl_model);
+
+  /// Byte swap the buffers field of a TFLite ModelT instance in place.
+  static void ByteSwapTFLiteModelT(tflite::ModelT* tfl_modelt);
+
+  /// Convert the TFLite buffers field between LE and BE format in a 
```

Comment:
This file has some unnecessary ending white spaces. Could you remove all?

Reply:
Sure

Reply:
Hi @terryheo ,

I've used `clang-format` to remove all unnecessary ending white spaces from this file. Also I've fixed another format issue in `tensorflow/lite/tools/serialization/BUILD` according to the `Code Check - Changed Files` results.

Could you please take a look?  Thank you very much!